spec/index.html

```diff
@@ -418,10 +418,35 @@ <h2>SPARQL Protocol Operations</h2>
         <li>The message content included in the HTTP response body.</li>
       </ul>
       <p>The SPARQL 1.2 Protocol is built on top of HTTP. All HTTP requirements for requests and responses <strong>must</strong> be followed.</p>
+
+     <section id="section-version-announcement">
+       <h3>Version Announcement</h3>
+
+       <p>When sending a query or update operation to an endpoint, the version of this operation MAY be announced by either using
+       the syntactical <a data-cite="SPARQL12-QUERY#syntaxVersionAnnouncement">`version` directive</a>,
+       or by passing a <code>version</code> parameter (which depends on the used HTTP method).</p>
+
+       <p>When the version is indicated both in syntax and a parameter,
+         they are expected to be the same.
+         If they differ,
+         parsers use the version from the parameter and might emit a warning about the mismatch.</p>
+
+       <p>To retain compability that is as broad as possible with older parsers,
+         only queries that make use of SPARQL 1.2-specific functionality
+         are encouraged to announce their version (i.e., for queries that do not make use
+         of SPARQL 1.2-specific functionality it is discouraged to announce a version).</p>
+
+       <p>Responses to an operation MAY also contain version announcements.
+       These <em>SPARQL results or RDF versions</em> are not necessarily the same as the <em>operation version</em> specified in the request.
+       Instead, the version announcement is dependent on the response and its format, such as [[[SPARQL12-RESULTS-JSON]]] or [[[RDF12-TURTLE]]].</p>
+
+       <p>Possible values for the version string are discussed in <a data-cite="RDF12-CONCEPTS#defined-version-labels"></a>.</p>
+      </section>
+
       <section id="query-operation">
         <h3>Query Operation</h3>
         <p>The <code>query</code> operation is used to send a SPARQL query to a service and receive the results of the query. The query operation <strong>MUST</strong> be invoked with either the HTTP
```

Comment:
```suggestion
        <p>The <code>query</code> operation is used to send a SPARQL query to a service and receive the results of the query. The query operation <strong>must</strong> be invoked with either the HTTP
```

Reply:
There is one existsing use of "MUST NOT" in section  query via GET.

Could you please align that as well?

Bonus points if fixing "1.1 Document Conventions".